build/depends.py

```diff
@@ -23,6 +23,17 @@ def configure(self, build, conf):
 
     def sources(self, build):
         return ['soundio/sounddeviceportaudio.cpp']
+        
+        
+class FilePathURL(Dependence):
```

Comment:
"FilePathURL" is a very generic name for a platform-specific and version-dependent workaround. How about renaming this to "OSXFilePathUrl" or "OSXFilePathUrlBackport"?